### PR TITLE
[WIP] week_05 first pass

### PR DESCRIPTION
**Pull request by choldgraf**
Hey @lwasser , here's the current state of the week 5 material. I haven't made any attempts to remove R code or change wording etc, I've basically only worked to recreate what (I think) the instructions wanted to plot with python. I definitely might have mis-interpreted some stuff so we should be iterating back and forth on this to make sure I'm not implementing something totally weird.

We can use this PR to go through drafts of the material, make improvements, etc. Once we think that all of the python outputs look correct, I can remove the R material.

# Current state

Most of the files are fully reproduced with python, still need to finish the last lecture + the homework.

### CODE REVIEW

**Comment by lwasser** [comment]
hey @choldgraf ok - i'll have a look at this!! 
Not to push you but where are you in terms of the markdown outputs - ie can we build these files yet? or will that be next week?

I"ll go through these in the next day and maybe if you haven't - you could start on the build part? i'm happy to help with translating the text in the files to python.

**Comment by lwasser** [comment]
Thank you!! i'm so excited to see this moving forward. 
I'm going to open an issue about rasterio and geopandas - i figured out the issues... and will include this with the setup!

**Comment by choldgraf** [comment]
> Not to push you but where are you in terms of the markdown outputs - ie can we build these files yet? or will that be next week?

Which do you think it's better to spend my time on? Markdown output + jekyll build, or converting R to python?

**Comment by lwasser** [comment]
hmmmm. good question @choldgraf and good point. we had talked about getting that nailed down in the first few weeks so it would be good to  figure that out BUT i'm ok if you'd like to work on that next month rather than this month. We had it being figured out in the next few weeks in the schedule.

**Comment by lwasser** [comment]
@choldgraf what's the best way for me to actually look at these notebooks? the diff's are not pretty!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
 # earth-analytics-python
 Space where we convert earth analytics class in R to  python
+
+# Installation
+
+We recommend installing geo-related dependencies with `conda-forge`. Use the following command on a **fresh** python installation:
```
Comment (lwasser) [inline README.md]: i found conda forge caused problems. i installed with pip and conda -- what issues did you run into with pip and conda?

**Comment by choldgraf** [comment]
> what's the best way for me to actually look at these notebooks? the diff's are not pretty

I'd do `Files changes -> view`. That'll show the full notebook (not the diff but still should be commentable).

re: what to work on, I think maybe we should try to get a jekyll build working. Does the website build already? AKA, is it just a matter of adding RST files at this point?

**Comment by lwasser** [comment]
that worked!! thank you :)

**Comment by lwasser** [comment]
i'll work on adding the build files. is it ok @choldgraf if i change directory names too? i'd like to change "templates" weeks or something more explicit.

**Comment by choldgraf** [comment]
sure, just so long as it's clear what is what

**Comment by lwasser** [comment]
great! :)

**Comment by lwasser** [comment]
hey @choldgraf starting to work through this. a few notes

data management is super important in this class so i have them working in an earth-analytics directory in their documents dir. i'd like to have all lessons stick to that path which means changing the WD for each lesson to be that path. Will you please work that way as well? I see a path that has a lot of ../../ in it!

Also - the diff's on these notebooks are going to be really difficult to manage. i'd love to be able to comment on lines. Is there any other option for us ?? i'm not set on these being notebooks but obviously it makes sense that they are.

**Comment by lwasser** [comment]
To be more clear they have an earth-analytics/data folder. then data for each week...

**Comment by lwasser** [comment]
so rather than this

sjer_plot_locations = gpd.read_file(op.join(path_data, 'california', 'SJER', 'vector_data', 'SJER_plot_centroids.shp'))

i'd rather have them gpd.read_file ("filename") ... rather than having to join together a path unless you think it's better to do the join. in R i have them just tab complete the path given a proper working directory

**Comment by lwasser** [comment]
ok so here is the code i'd prefer for example lessons 01 or week 5

`leahs_test = gpd.read_file(filename="data/week5/california/SJER/vector_data/SJER_plot_centroids.shp")`

**Pull request merged**